CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased](https://github.com/schemathesis/schemathesis/compare/v4.3.15...HEAD) - TBD
 
+### :wrench: Changed
```

Comment:
This change has no user facing effect - in such cases, I prefer not adding a changelog entry

Reply:
Delete it in Changelog.md